File: service-center-app/src/app/models/CustomerCar.ts
export class CustomerCar {

    vin: number;
    mileage: number;
    last_SCHEDULED_MAINTENANCE: number;
    customer_ID: number;
    service_CENTER_ID: number;
    car_ID: number;
    year: number;

    constructor(json?: any) {
        if (json) {
            this.vin = json.vin;
            this.mileage = json.mileage;
            this.last_SCHEDULED_MAINTENANCE = json.last_SCHEDULED_MAINTENANCE;
            this.customer_ID = json.customer_ID;
            this.service_CENTER_ID = json.service_CENTER_ID;
            this.car_ID = json.car_ID;
            this.year = json.year;
        }
        else{
            this.vin = 0;
            this.mileage = 0;
            this.last_SCHEDULED_MAINTENANCE = 0;
            this.customer_ID = 0;
            this.service_CENTER_ID = 0;
            this.car_ID = 0;
            this.year = 0;
        }
    }
}

File: src/main/java/com/dbms/team15/models/CustomerCar.java
package com.dbms.team15.models;

public class CustomerCar {

  private int VIN;
  private double MILEAGE;
  private int LAST_SCHEDULED_MAINTENANCE;
  private int CUSTOMER_ID;
  private int SERVICE_CENTER_ID;
  private int CAR_ID;

  public CustomerCar() {}

  public CustomerCar(
    int VIN,
    double MILEAGE,
    int LAST_SCHEDULED_MAINTENANCE,
    int CUSTOMER_ID,
    int SERVICE_CENTER_ID,
    int CAR_ID
  ) {
    this.VIN = VIN;
    this.MILEAGE = MILEAGE;
    this.LAST_SCHEDULED_MAINTENANCE = LAST_SCHEDULED_MAINTENANCE;
    this.CUSTOMER_ID = CUSTOMER_ID;
    this.SERVICE_CENTER_ID = SERVICE_CENTER_ID;
    this.CAR_ID = CAR_ID;
  }

  public int getVIN() {
    return VIN;
  }

  public double getMILEAGE() {
    return MILEAGE;
  }

  public int getLAST_SCHEDULED_MAINTENANCE() {
    return LAST_SCHEDULED_MAINTENANCE;
  }

  public int getCUSTOMER_ID() {
    return CUSTOMER_ID;
  }

  public int getSERVICE_CENTER_ID() {
    return SERVICE_CENTER_ID;
  }

  public int getCAR_ID() {
    return CAR_ID;
  }

  public void setVIN(int VIN) {
    this.VIN = VIN;
  }

  public void setMILEAGE(double MILEAGE) {
    this.MILEAGE = MILEAGE;
  }

  public void setLAST_SCHEDULED_MAINTENANCE(int LAST_SCHEDULED_MAINTENANCE) {
    this.LAST_SCHEDULED_MAINTENANCE = LAST_SCHEDULED_MAINTENANCE;
  }

  public void setCUSTOMER_ID(int CUSTOMER_ID) {
    this.CUSTOMER_ID = CUSTOMER_ID;
  }

  public void setSERVICE_CENTER_ID(int SERVICE_CENTER_ID) {
    this.SERVICE_CENTER_ID = SERVICE_CENTER_ID;
  }

  public void setCAR_ID(int CAR_ID) {
    this.CAR_ID = CAR_ID;
  }

  //To String
  @Override
  public String toString() {
    return (
      "CustomerCar{" +
      "VIN=" +
      VIN +
      ", MILEAGE=" +
      MILEAGE +
      ", LAST_SCHEDULED_MAINTENANCE=" +
      LAST_SCHEDULED_MAINTENANCE +
      ", CUSTOMER_ID=" +
      CUSTOMER_ID +
      ", SERVICE_CENTER_ID=" +
      SERVICE_CENTER_ID +
      ", CAR_ID=" +
      CAR_ID +
      '}'
    );
  }
}
